### ISSUE TITLE
Provided openal version causes all manner of problems

### ISSUE THREAD

**Issue by RomanHargrave** [ValveSoftware/steam-runtime#70]
I just spent about two days chasing the wrong thing in an attempt to fix the _incredibly_ awful audio in Victor Vran (neon chrome has the same problem, but I haven't tested it with this yet).

I use JACK on top of ALSA, and Pulse sits on top of JACK for application support, so I haven't tested these two games with Pulse as the software mixer instead of JACK.

With the SteamRT-provided version of libopenal (version 1.13 (wow)) these two (and probably more games) experience stuttering audio due to buffer underruns within pulse (that are not Pulse's fault). While trying to make VV connect to JACK instead of Pulse, I built a copy of OpenAL at `5a50c46c`. While this still didn't resolve the issues with JACK due to some odd business that apparently isn't in OpenAL (which I suspect is in VV, which I also suspect has obfuscated strings - naughty!), I did eventually try starting the game with the new OpenAL version and using pulse as the backend - which fixed all issues I had.

**TL;DR** - the provided OpenAL is causing serious issues with some games.

**Comment by kisak-valve** [comment]
Closing as there already is a request to update openal in the steam runtime at #18.
